trunk/mailer.http.class.php

```diff
@@ -75,10 +75,7 @@ protected function get_request_body()
                 break;
         }
 
-        $replyTos = $this->getReplyToAddresses();
-        if (count($replyTos)) {
-            $body['content']['reply_to'] =  $replyTos[0];
-        }
+        $body['content'] = array_merge($body['content'], $this->get_headers_in_content());
```

Comment:
I don't like the name now. Suggest me a better name. may be `get_headers_for_content`?